risc0/zkvm/Cargo.toml

```diff
@@ -63,6 +63,7 @@ addr2line = { version = "0.21", features = ["rustc-demangle"], optional = true }
 bincode = { version = "1.3", optional = true }
 bonsai-sdk = { workspace = true, optional = true }
 bytes = { version = "1.4", features = ["serde"], optional = true }
+clap = { version = "4.4", features = ["derive"] }
```

Comment:
Is this needed?

Reply:
This is needed for r0vm because r0vm needs this (and other fields of prover Opts) as a command line argument. I could create a separate struct within r0vm that's meant to capture this information and translate that to `ReceiptKind` which would allow us to drop this dependency. Would that be preferred?

Reply:
Why wouldn't this go on the `risc0-r0vm` crate? I don't see why `risc0-zkvm` needs it.

Reply:
It was for derive `ValueEnum` that I had placed. I pushed a change that removes this derive, drops clap and created a new struct in r0vm. i feel like it's probably a better approach to keep the clap stuff away from risc0-zkvm